src/main/kotlin/com/wire/bots/infrastructure/client/BearerTokenParamConverter.kt

```diff
@@ -34,9 +32,13 @@ class BearerTokenParamConverterProvider : ParamConverterProvider {
     override fun <T> getConverter(
         rawType: Class<T>,
         genericType: Type?,
-        annotations: Array<Annotation?>?,
+        annotations: Array<Annotation?>?
     ): ParamConverter<T>? {
-        if (rawType == String::class.java) return safeCast<ParamConverter<T>>(BearerTokenParamConverter())
+        if (rawType ==
+            String::class.java
+        ) {
+            return safeCast<ParamConverter<T>>(BearerTokenParamConverter())
+        }
```

Comment:
Shouldn't the condition be on the same line as the length of the line is sufficient? 

```suggestion
        if (rawType == String::class.java) {
            return safeCast<ParamConverter<T>>(BearerTokenParamConverter())
        }
```